chart/values.yaml

```diff
@@ -153,3 +153,8 @@ promlens:
   defaultPrometheusUrl:  "http://localhost:9201"
   loadBalancer:
     enabled: false
+
+#Enable Promscale to connect to an existing database by providing the db-uri
+external_database:
```

Comment:
i would rename this to `timescaledb-external` and put above timescaledb-single

Reply:
Having the name with `-` making the chart complex while accessing the fields. [Here](https://github.com/helm/helm/issues/2192) is a discussion on the same in helm repo. So renaming it to camelcase i.e. `timescaledbExternal`